README.md

```diff
@@ -687,19 +745,7 @@ Wrong:
 ```php
 <?php
 
-function find($needle, $haystack)
```

Comment:
Why this example was removed?

Reply:
IMO we should keep it as example is also anti pattern - result should be returned as soon as possible for maximum efficiency. I guess later on separate sections for antipatterns can be added.